buffalo/cmd/app_generators.go

```diff
@@ -361,3 +366,18 @@ ratings:
     - "**.go"
 
 `
+
+const nTravis = `language: go
+env:
+- GO_ENV=test
+
+before_script:
```

Comment:
this raises an interesting question about databases here. this is specific to Postgres, but what if they generate with mysql or sqlite?